python/pyarrow/feather.py

```diff
@@ -69,10 +69,8 @@ def read_table(self, columns=None):
 
     def validate_schemas(self, piece, table):
         if not self.schema.equals(table.schema):
-            raise ValueError('Schema in {!s} was different. \n'
-                             '{!s}\n\nvs\n\n{!s}'
-                             .format(piece, self.schema,
-                                     table.schema))
+            raise ValueError(f'Schema in {piece!s} was different. \n'
+                             f'{self.schema!s}\n\nvs\n\n{table.schema!s}')
```

Comment:
I think `!s` is generally implicit.

Reply:
Resolved in [c926dde](https://github.com/apache/arrow/pull/45629/commits/c926ddec73bfdeb7d6c3b3768ade16f8146eadd4)